Resources/config/security_acl.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Honestly I don't dare messing with the services ids. I've never used the ACL features myself so I can't really predict what would happen